addons/binding/org.openhab.binding.lgwebos/src/main/java/org/openhab/binding/lgwebos/internal/discovery/LGWebOSUpnpDiscoveryParticipant.java.bak

```diff
@@ -0,0 +1,81 @@
+package org.openhab.binding.lgwebos.internal.discovery;
```

Comment:
I guess this file should be deleted?

Reply:
Oh, this should have never been pushed. Will be deleted.